File: testprojects/python_main/pipelines/PYTHON_UNITY_CATALOG/code/python_unity_catalog/graph/all_type_parquet_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from python_unity_catalog.config.ConfigStore import *
from python_unity_catalog.udfs.UDFs import *

def all_type_parquet_1(spark: SparkSession) -> DataFrame:
    return spark.read.table(f"qa_database.all_type_parquet")

File: testprojects/scala_main/pipelines/SCALA_UNITY_CATALOG/code/src/main/scala/io/prophecy/pipelines/scala_unity_catalog/graph/all_type_parquet_1.scala
package io.prophecy.pipelines.scala_unity_catalog.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scala_unity_catalog.config.ConfigStore._
import io.prophecy.pipelines.scala_unity_catalog.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object all_type_parquet_1 {

  def apply(context: Context): DataFrame =
    context.spark.read.table("qa_database.all_type_parquet")

}
